sorbet/tapioca/config.yml

```diff
@@ -1,2 +1,4 @@
 ---
 exclude: []
+typed_overrides:
+  "pry-doc": "false"
```

Comment:
`Pry-doc` re-defines a bunch of methods from other pry-related libraries and type checking fails.